network_analysis/find-service-areas-for-multiple-facilities/README.md

```diff
@@ -0,0 +1,38 @@
+# Find service areas for multiple facilities
+
+Find the service areas of several facilities from a feature service.
+
+![](FindServiceAreasForMultipleFacilities.png)
+
+## Use case
+
+A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be served effectively from particular hospitals, and with this optimize distribution of staff and resources.
+
+## How to use the sample
+
+Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represents the facility's service area; the red area is within 1 minute travel time from the hospital by car, whilst orange is within 3 minutes by car. All polygons are transparent, so that when service areas from multiple facilities overlap, these areas will appear as bolder colors.
```

Comment:
whilst -> wheras or while

Reply:
> All polygons are transparent, so that when service areas from multiple facilities overlap, these areas will appear as bolder colors.

All service areas are semi-transparent to show where they overlap.